[2023-10-01 00:05:14,530] {scheduler_job.py:182} INFO - Started process (PID=98) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:14,537] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:05:14,557] {logging_mixin.py:104} INFO - [2023-10-01 00:05:14,556] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:14,789] {logging_mixin.py:104} INFO - [2023-10-01 00:05:14,758] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:05:14,805] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:14,886] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.382 seconds
[2023-10-01 00:05:45,890] {scheduler_job.py:182} INFO - Started process (PID=150) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:45,900] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:05:45,905] {logging_mixin.py:104} INFO - [2023-10-01 00:05:45,904] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:46,061] {logging_mixin.py:104} INFO - [2023-10-01 00:05:46,048] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:05:46,070] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:05:46,156] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.281 seconds
[2023-10-01 00:06:16,606] {scheduler_job.py:182} INFO - Started process (PID=204) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:16,612] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:06:16,617] {logging_mixin.py:104} INFO - [2023-10-01 00:06:16,616] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:16,756] {logging_mixin.py:104} INFO - [2023-10-01 00:06:16,735] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:06:16,762] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:16,843] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.249 seconds
[2023-10-01 00:06:46,929] {scheduler_job.py:182} INFO - Started process (PID=258) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:46,945] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:06:46,950] {logging_mixin.py:104} INFO - [2023-10-01 00:06:46,949] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:47,140] {logging_mixin.py:104} INFO - [2023-10-01 00:06:47,124] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:06:47,150] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:06:47,245] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.336 seconds
[2023-10-01 00:07:18,673] {scheduler_job.py:182} INFO - Started process (PID=314) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:18,690] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:07:18,701] {logging_mixin.py:104} INFO - [2023-10-01 00:07:18,700] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:18,881] {logging_mixin.py:104} INFO - [2023-10-01 00:07:18,866] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:07:18,895] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:19,005] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.357 seconds
[2023-10-01 00:07:49,535] {scheduler_job.py:182} INFO - Started process (PID=366) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:49,551] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:07:49,559] {logging_mixin.py:104} INFO - [2023-10-01 00:07:49,558] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:49,776] {logging_mixin.py:104} INFO - [2023-10-01 00:07:49,763] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:07:49,782] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:07:49,845] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.327 seconds
[2023-10-01 00:08:20,620] {scheduler_job.py:182} INFO - Started process (PID=420) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:20,626] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:08:20,632] {logging_mixin.py:104} INFO - [2023-10-01 00:08:20,631] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:20,716] {logging_mixin.py:104} INFO - [2023-10-01 00:08:20,708] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:08:20,722] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:20,773] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.165 seconds
[2023-10-01 00:08:52,349] {scheduler_job.py:182} INFO - Started process (PID=474) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:52,369] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:08:52,380] {logging_mixin.py:104} INFO - [2023-10-01 00:08:52,378] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:52,729] {logging_mixin.py:104} INFO - [2023-10-01 00:08:52,704] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:08:52,739] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:08:52,894] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.588 seconds
[2023-10-01 00:09:23,121] {scheduler_job.py:182} INFO - Started process (PID=528) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:23,126] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:09:23,130] {logging_mixin.py:104} INFO - [2023-10-01 00:09:23,129] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:23,205] {logging_mixin.py:104} INFO - [2023-10-01 00:09:23,199] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:09:23,211] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:23,258] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.146 seconds
[2023-10-01 00:09:53,512] {scheduler_job.py:182} INFO - Started process (PID=582) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:53,517] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:09:53,521] {logging_mixin.py:104} INFO - [2023-10-01 00:09:53,520] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:53,603] {logging_mixin.py:104} INFO - [2023-10-01 00:09:53,590] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from Airflow.airflow.dags.extract import extract
ModuleNotFoundError: No module named 'Airflow'
[2023-10-01 00:09:53,608] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:09:53,644] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.141 seconds
[2023-10-01 00:10:24,260] {scheduler_job.py:182} INFO - Started process (PID=636) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:24,269] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:10:24,278] {logging_mixin.py:104} INFO - [2023-10-01 00:10:24,277] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:24,405] {logging_mixin.py:104} INFO - [2023-10-01 00:10:24,397] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:10:24,412] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:24,462] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.215 seconds
[2023-10-01 00:10:54,843] {scheduler_job.py:182} INFO - Started process (PID=690) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:54,850] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:10:54,854] {logging_mixin.py:104} INFO - [2023-10-01 00:10:54,853] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:54,977] {logging_mixin.py:104} INFO - [2023-10-01 00:10:54,964] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:10:54,990] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:10:55,047] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.216 seconds
[2023-10-01 00:12:36,322] {scheduler_job.py:182} INFO - Started process (PID=89) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:12:36,343] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:12:36,356] {logging_mixin.py:104} INFO - [2023-10-01 00:12:36,355] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:12:36,653] {logging_mixin.py:104} INFO - [2023-10-01 00:12:36,629] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:12:36,666] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:12:36,880] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.603 seconds
[2023-10-01 00:13:07,082] {scheduler_job.py:182} INFO - Started process (PID=141) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:07,103] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:13:07,116] {logging_mixin.py:104} INFO - [2023-10-01 00:13:07,115] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:07,406] {logging_mixin.py:104} INFO - [2023-10-01 00:13:07,380] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:13:07,417] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:07,513] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.467 seconds
[2023-10-01 00:13:38,150] {scheduler_job.py:182} INFO - Started process (PID=195) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:38,161] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:13:38,172] {logging_mixin.py:104} INFO - [2023-10-01 00:13:38,170] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:38,389] {logging_mixin.py:104} INFO - [2023-10-01 00:13:38,370] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:13:38,402] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:13:38,565] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.427 seconds
[2023-10-01 00:14:08,915] {scheduler_job.py:182} INFO - Started process (PID=247) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:08,929] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:14:08,939] {logging_mixin.py:104} INFO - [2023-10-01 00:14:08,938] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:09,160] {logging_mixin.py:104} INFO - [2023-10-01 00:14:09,142] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:14:09,172] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:09,247] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.366 seconds
[2023-10-01 00:14:39,674] {scheduler_job.py:182} INFO - Started process (PID=301) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:39,695] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:14:39,712] {logging_mixin.py:104} INFO - [2023-10-01 00:14:39,704] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:39,983] {logging_mixin.py:104} INFO - [2023-10-01 00:14:39,960] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:14:39,996] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:14:40,108] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.486 seconds
[2023-10-01 00:15:10,251] {scheduler_job.py:182} INFO - Started process (PID=353) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:10,267] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:15:10,273] {logging_mixin.py:104} INFO - [2023-10-01 00:15:10,272] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:10,431] {logging_mixin.py:104} INFO - [2023-10-01 00:15:10,407] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:15:10,440] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:10,530] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.301 seconds
[2023-10-01 00:15:40,872] {scheduler_job.py:182} INFO - Started process (PID=407) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:40,885] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:15:40,891] {logging_mixin.py:104} INFO - [2023-10-01 00:15:40,890] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:40,992] {logging_mixin.py:104} INFO - [2023-10-01 00:15:40,983] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:15:41,003] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:15:41,056] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.198 seconds
[2023-10-01 00:16:11,263] {scheduler_job.py:182} INFO - Started process (PID=461) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:16:11,270] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:16:11,274] {logging_mixin.py:104} INFO - [2023-10-01 00:16:11,273] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:16:11,394] {logging_mixin.py:104} INFO - [2023-10-01 00:16:11,387] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from airflow.dags.extract import extract
ModuleNotFoundError: No module named 'airflow.dags'
[2023-10-01 00:16:11,401] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:16:11,442] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.190 seconds
[2023-10-01 00:17:30,813] {scheduler_job.py:182} INFO - Started process (PID=88) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:17:30,826] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:17:30,835] {logging_mixin.py:104} INFO - [2023-10-01 00:17:30,831] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:17:31,203] {logging_mixin.py:104} INFO - [2023-10-01 00:17:31,193] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:17:31,214] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:17:31,321] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.549 seconds
[2023-10-01 00:18:02,358] {scheduler_job.py:182} INFO - Started process (PID=141) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:02,377] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:18:02,391] {logging_mixin.py:104} INFO - [2023-10-01 00:18:02,385] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:02,747] {logging_mixin.py:104} INFO - [2023-10-01 00:18:02,715] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:18:02,774] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:02,943] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.633 seconds
[2023-10-01 00:18:33,381] {scheduler_job.py:182} INFO - Started process (PID=194) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:33,394] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:18:33,414] {logging_mixin.py:104} INFO - [2023-10-01 00:18:33,404] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:33,719] {logging_mixin.py:104} INFO - [2023-10-01 00:18:33,691] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:18:33,730] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:18:33,817] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.482 seconds
[2023-10-01 00:19:04,258] {scheduler_job.py:182} INFO - Started process (PID=248) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:04,275] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:19:04,289] {logging_mixin.py:104} INFO - [2023-10-01 00:19:04,288] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:04,487] {logging_mixin.py:104} INFO - [2023-10-01 00:19:04,472] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:19:04,496] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:04,565] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.333 seconds
[2023-10-01 00:19:34,811] {scheduler_job.py:182} INFO - Started process (PID=302) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:34,827] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:19:34,835] {logging_mixin.py:104} INFO - [2023-10-01 00:19:34,835] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:35,125] {logging_mixin.py:104} INFO - [2023-10-01 00:19:35,101] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:19:35,139] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:19:35,236] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.462 seconds
[2023-10-01 00:20:05,482] {scheduler_job.py:182} INFO - Started process (PID=354) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:05,495] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:20:05,501] {logging_mixin.py:104} INFO - [2023-10-01 00:20:05,500] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:05,796] {logging_mixin.py:104} INFO - [2023-10-01 00:20:05,759] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:20:05,809] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:05,947] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.484 seconds
[2023-10-01 00:20:36,194] {scheduler_job.py:182} INFO - Started process (PID=408) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:36,206] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:20:36,212] {logging_mixin.py:104} INFO - [2023-10-01 00:20:36,211] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:36,395] {logging_mixin.py:104} INFO - [2023-10-01 00:20:36,380] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:20:36,403] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:20:36,485] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.313 seconds
[2023-10-01 00:21:07,065] {scheduler_job.py:182} INFO - Started process (PID=460) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:07,077] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:21:07,088] {logging_mixin.py:104} INFO - [2023-10-01 00:21:07,087] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:07,344] {logging_mixin.py:104} INFO - [2023-10-01 00:21:07,323] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:21:07,353] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:07,451] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.423 seconds
[2023-10-01 00:21:38,036] {scheduler_job.py:182} INFO - Started process (PID=512) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:38,061] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:21:38,071] {logging_mixin.py:104} INFO - [2023-10-01 00:21:38,070] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:38,253] {logging_mixin.py:104} INFO - [2023-10-01 00:21:38,238] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:21:38,259] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:21:38,311] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.301 seconds
[2023-10-01 00:22:08,696] {scheduler_job.py:182} INFO - Started process (PID=566) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:08,709] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:22:08,718] {logging_mixin.py:104} INFO - [2023-10-01 00:22:08,716] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:08,932] {logging_mixin.py:104} INFO - [2023-10-01 00:22:08,913] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:22:08,943] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:09,091] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.411 seconds
[2023-10-01 00:22:39,424] {scheduler_job.py:182} INFO - Started process (PID=618) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:39,450] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:22:39,486] {logging_mixin.py:104} INFO - [2023-10-01 00:22:39,483] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:39,764] {logging_mixin.py:104} INFO - [2023-10-01 00:22:39,745] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:22:39,781] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:22:39,842] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.458 seconds
[2023-10-01 00:23:09,909] {scheduler_job.py:182} INFO - Started process (PID=670) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:09,915] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:23:09,919] {logging_mixin.py:104} INFO - [2023-10-01 00:23:09,918] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:10,001] {logging_mixin.py:104} INFO - [2023-10-01 00:23:09,992] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:23:10,005] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:10,042] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.143 seconds
[2023-10-01 00:23:40,452] {scheduler_job.py:182} INFO - Started process (PID=726) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:40,460] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:23:40,466] {logging_mixin.py:104} INFO - [2023-10-01 00:23:40,466] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:40,586] {logging_mixin.py:104} INFO - [2023-10-01 00:23:40,572] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:23:40,593] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:23:40,652] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.213 seconds
[2023-10-01 00:24:10,801] {scheduler_job.py:182} INFO - Started process (PID=778) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:10,808] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:24:10,813] {logging_mixin.py:104} INFO - [2023-10-01 00:24:10,812] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:10,904] {logging_mixin.py:104} INFO - [2023-10-01 00:24:10,892] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:24:10,913] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:10,955] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.164 seconds
[2023-10-01 00:24:41,474] {scheduler_job.py:182} INFO - Started process (PID=832) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:41,484] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:24:41,488] {logging_mixin.py:104} INFO - [2023-10-01 00:24:41,487] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:41,584] {logging_mixin.py:104} INFO - [2023-10-01 00:24:41,571] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:24:41,589] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:24:41,629] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.164 seconds
[2023-10-01 00:25:13,085] {scheduler_job.py:182} INFO - Started process (PID=888) to work on /opt/airflow/dags/extract_dags.py
[2023-10-01 00:25:13,090] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/extract_dags.py for tasks to queue
[2023-10-01 00:25:13,094] {logging_mixin.py:104} INFO - [2023-10-01 00:25:13,094] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:25:13,202] {logging_mixin.py:104} INFO - [2023-10-01 00:25:13,188] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/extract_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/extract_dags.py", line 10, in <module>
    from extract import extract
  File "/opt/airflow/dags/extract.py", line 5, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2023-10-01 00:25:13,207] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/extract_dags.py
[2023-10-01 00:25:13,251] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/extract_dags.py took 0.178 seconds
